outputs.tf

```diff
@@ -84,6 +69,37 @@ output "kubernetes_secrets" {
   })
 }
 
+output "helm_values" {
```

Comment:
Can we maybe output the security group policy too?

Reply:
We already do.

Reply:
right, the one spacectl messed up